third_party/silabs/SiWx917_sdk.gni

```diff
@@ -772,10 +773,8 @@ template("siwx917_sdk") {
       "${efr32_sdk_root}/platform/service/iostream/src/sl_iostream.c",
       "${efr32_sdk_root}/platform/service/iostream/src/sl_iostream_rtt.c",
 
-      # STARTUP FILES
-      # TODO swap to startup_si91x and new linkerfile once Board_Debug_Init is fixed.
-      #"${wifi_sdk_root}/components/device/silabs/si91x/mcu/core/chip/src/startup_si91x.c",
-      "${matter_support_root}/board-support/si91x/support/src/startup_common_RS1xxxx.c",
+      # STARTUP FILES.
+      "${wifi_sdk_root}/components/device/silabs/si91x/mcu/core/chip/src/startup_si91x.c",
```

Comment:
can we delete the files from matter_support? Not sure if the previous PR did this.

Reply:
yes, I did open a pr to delete those already

Reply:
ah, missed the PR since it was not logged in.